packages/cicero-core/test/metadata.js

```diff
@@ -113,6 +113,14 @@ describe('Metadata', () => {
             }, {}, {})).should.throw('README must be a string');
         });
 
+        it('should throw an error if logo is not a string', () => {
```

Comment:
The error doesn't seem to be consistent with the name of the test. `logo` should be a `Buffer` not a string.

Reply:
@dselman Yeah, I know. I didn't know what data type to use initially so I just wrote what `loadFileContents` was returning :sweat_smile: 
I will submit the changes as soon as possible.